commonui/src/test/java/com/hyperwallet/android/ui/common/util/TransferMethodUtilsTest.java

```diff
@@ -74,15 +75,15 @@ public void testGetTransferMethodName_returnsCorrectNameForType() throws JSONExc
         HyperwalletTransferMethod hyperwalletTransferMethod = new HyperwalletTransferMethod(htmJsonObject);
         when(mContext.getString(R.string.bank_card)).thenReturn("My card");
         String transferMethodName = TransferMethodUtils.getTransferMethodName(mContext, hyperwalletTransferMethod);
-        assertThat(transferMethodName, is("My card"));
+        MatcherAssert.assertThat(transferMethodName, Matchers.is("My card"));
         verify(mContext, times(1)).getString(anyInt());
     }
 
     @Test
     public void testGetTransferMethodName_returnsDefaultValue() {
         when(mContext.getString(R.string.not_translated_in_braces)).thenReturn("(not translated)");
         String transferMethodName = TransferMethodUtils.getTransferMethodName(mContext, "some custom name");
-        assertThat(transferMethodName, is("some custom name(not translated)"));
+        MatcherAssert.assertThat(transferMethodName, Matchers.is("some custom name(not translated)"));
```

Comment:
please import the whole static class for `MatcherAssert` and `Matchers` please

Reply:
remove